org.getmonero.Monero.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: org.getmonero.Monero
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --share=ipc
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: monero-wallet-gui
+modules:
+  - name: monero-bin
+    buildsystem: simple
+    sources:
+    - type: archive
+      url: https://downloads.getmonero.org/gui/monero-gui-linux-x64-v0.17.2.0.tar.bz2
+      sha256: ee31d67f3df174cc88b01a528aa9d882504fb5f7ae805f449ff35820a4d45bbb
+    build-commands:
+    - install -d /app/bin
```

Comment:
You might want to set the executable-bit (or however that's called) on those files:

```yaml
- install --directory --mode=0755 /app/bin/
- install -D --mode=0755 --target-directory=/app/bin/ monero-wallet-gui
- install -D --mode=0755 --target-directory=/app/bin/ monerod
```

Also, I would simplify the manifest and put the other sources in the same module:

```yaml
  ...
  - install -D --mode=0644 --target-directory=/app/share/applications/ org.getmonero.Monero.desktop
  - install -D --mode=0644 --target-directory=/app/share/icons/hicolor/scalable/apps/ org.getmonero.Monero.svg
sources:
  - type: archive
    sha256: 0088b133febdb151a3753393b4dc8b06111b844c0fca14e512f3c8d56a2a5b97
    url: https://github.com/monero-project/monero-gui/releases/download/v0.17.2.2/monero-gui-linux-x64-v0.17.2.2.tar.bz2
  - type: file
    path: org.getmonero.Monero.desktop
  - type: file
    path: org.getmonero.Monero.svg
```